CONTRIBUTING.md

```diff
@@ -11,7 +11,8 @@ We use the github [fork and pull review process](https://help.github.com/article
 
 Add an upstream remote so you can easily keep up to date with the main repository:
 
-git remote add upstream https://github.com/google/timesketch.git
+``` $ git remote add upstream https://github.com/google/timesketch.git ```
```

Comment:
Can you change this to be 4 spaces instead? Just to stay consistent with the rest of the file.

Reply:
Changed to 4 spaces